About.html

```diff
@@ -0,0 +1,107 @@
+    <title>About - Darko</title>
+</head>
+<body>
+    <main>
+        <div class="socials">
+            <ul class="social-m">
+                <li><a href="#" class="nono facebook-to"><i class="fab fa-facebook-f"></i></a></li>
+                <li><a href="#" class="nono twitter-to"><i class="fab fa-twitter"></i></a></li>
+                <li><a href="#" class="nono"><p class="fab">English</p></a></li>
+                <li><a href="#" class="nono"><p class="fab">Log out</p></a></li>
+            </ul>
+        </div>
+        <nav class="nav-bar">
+            <span class="btn humberger-btn"><i class="fas fa-bars" onclick="changeIcon(this)"></i></span>
+            <label class="logo"><a href="index.html" class="refr"><img src="IMAGES/darko-logo.png" alt="logo" class="logo-desk"></a></label>
+            <ul class="menu">
+                <li class="menu-choice home"><a href="index.html" class="link-desk">Home</a></li>
+                <li class="menu-choice about"><a href="About.html" class="link-desk">About</a></li>
+                <li class="menu-choice blog"><a href="" class="link-desk">Blog</a></li>
+                <li class="menu-choice news"><a href="" class="link-desk">News</a></li>
+                <li class="menu-choice sponsor"><a href="" class="link-desk">Sponsor</a></li>
+                <li class="menu-choice call" id="contact"><a href="" class="link-desk link-desko">Call Us</a></li>
+            </ul>
+        </nav>
+        <ul class="options">
+            <li class="choice"><a href="index.html" class="go-to-page">Home</a></li>
+            <li class="choice "><a href="About.html" class="go-to-page">About</a></li>
+            <li class="choice"><a href="#" class="go-to-page">Blog</a></li>
+            <li class="choice"><a href="#" class="go-to-page">Sponsor</a></li>
+            <li class="choice"><a href="#" class="go-to-page">News</a></li>
+            <li class="choice"><a href="#" class="go-to-page">Contact</a></li>
+        </ul>
+        <section class="summary">
+            <p class="greeting">Hello! Future Musicians,</p>
+            <div class="a-title"><h1 class="summary-title">DARKO MUSIC COURSE 2022 SUMMER INTAKE</h1></div>
+            <div class="in-short">
+                <p class="text">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Possimus explicabo nemo laboriosam minima alias, reiciendis a! Vel iste cupiditate provident, inventore exercitationem, recusandae quidem dicta dolores soluta aut aspernatur ipsum!</p>
+            </div>
+            <div class="email-us">
+                <p class="text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Porro veritatis sapiente quam, alias quod pariatur.</p>
+                <p class="our-email">darkomusic@rda.org</p>
+            </div>
+        </section>
+        <section class="hist-sect">
+            <div class="hist-title">
+                <h2 class="real-title">Darko Music 2022 History</h2>
+            </div>
```

Comment:
- [ ] Kindly move this header more to the bottom because it has to be on the second section and right now it has the background of the first section.

Here you have a screenshot from your project:

![image](https://user-images.githubusercontent.com/74506933/163674574-51698c48-bfbe-4c64-89bc-a5d4435c39f0.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/163674588-ee727026-4c2c-41f1-817a-0aa90fa39e78.png)